pkg/providers/ovirt/validation/validators/network-mapping-validator.go

```diff
@@ -89,6 +98,24 @@ func (v *NetworkMappingValidator) hasAtLeastOneWithVNicProfile(nics []*ovirtsdk.
 	return false
 }
 
+func (v *NetworkMappingValidator) validateSourceNetworkFormat(mapByName map[string]v2vv1alpha1.ResourceMappingItem) (ValidationFailure, bool) {
+	invalidNames := make([]string, 0)
+	for k := range mapByName {
+		if !strings.Contains(k, "/") {
```

Comment:
Is it possible that either network name or vnicprofile name could contain `\`?

Reply:
it seems that both might contain any set of characters - the restriction for network/vnic name was removed https://gerrit.ovirt.org/#/c/79459/ 
it even seems legal for vnic profile name or network name to contain only `/////` 

However, that doesn't break the semantic of this field. If there is no `/` - it means it doesn't contain both elements. If it contains at least `/` and it doesn't match any of vm nics - it should fail.

The alternative is suggesting a new element to identify the network name, so the element will look like as:
```
networkMappings:
   source:
      vnicProfileName: profile1
      networkName: red
      vnicProfileID: xxx-xxx-xxx
```
I'd prefer staying with current format of a common `ResourceMappingItem` to identify sources by ID or name and not suggest element for each source.